apis/pkg/meta/v1alpha1/configuration_types.go

```diff
@@ -34,3 +43,61 @@ type Configuration struct {
 
 	Spec ConfigurationSpec `json:"spec"`
 }
+
```

Comment:
The below conversion functions are new. They're identical to the equivalent v1 conversion functions.